src/CPUSolver.cpp

```diff
@@ -859,40 +859,40 @@ void CPUSolver::addSourceToScalarFlux() {
 
 /**
  * @brief Computes the volume-integrated, energy-integrated nu-fission rate in
- *        each FSR and stores them in an array indexed by FSR ID.
+ *        each SR and stores them in an array indexed by SR ID.
  * @details This is a helper method for SWIG to allow users to retrieve
- *          FSR nu-fission rates as a NumPy array. An example of how this
+ *          SR nu-fission rates as a NumPy array. An example of how this
  *          method can be called from Python is as follows:
  *
  * @code
- *          num_FSRs = geometry.getNumFSRs()
- *          fission_rates = solver.computeFSRFissionRates(num_FSRs)
+ *          num_SRs = geometry.getNumSRs()
+ *          fission_rates = solver.computeSRFissionRates(num_SRs)
  * @endcode
  *
  * @param fission_rates an array to store the nu-fission rates (implicitly
  *                      passed in as a NumPy array from Python)
- * @param num_FSRs the number of FSRs passed in from Python
+ * @param num_SRs the number of SRs passed in from Python
  */
-void CPUSolver::computeFSRFissionRates(double* fission_rates, int num_FSRs) {
+void CPUSolver::computeSRFissionRates(double* fission_rates, int num_SRs) {
 
   if (_scalar_flux == NULL)
-    log_printf(ERROR, "Unable to compute FSR fission rates since the "
+    log_printf(ERROR, "Unable to compute SR fission rates since the "
                "source distribution has not been calculated");
 
-  log_printf(INFO, "Computing FSR fission rates...");
+  log_printf(INFO, "Computing SR fission rates...");
 
   FP_PRECISION* sigma_f;
   FP_PRECISION volume;
 
   /* Initialize fission rates to zero */
-  for (int r=0; r < _num_FSRs; r++)
+  for (int r=0; r < _num_SRs; r++)
     fission_rates[r] = 0.0;
 
-  /* Loop over all FSRs and compute the volume-averaged fission rate */
+  /* Loop over all SRs and compute the volume-averaged nu-fission rate */
```

Comment:
Indeed. Glad you pointed out this detail of the compute fission sources procedure out!